package.json

```diff
@@ -1,7 +1,7 @@
 {
     "name": "mssql",
     "displayName": "SQL Server (mssql)",
-    "version": "1.33.0",
+    "version": "1.32.0",
```

Comment:
this PR shouldn't change the package.json.  The package.json in main is targeting June, which should be 1.33.  Release branch is still at 1.32.  But I need to move the changes, so let's just go with this and I'll fix up later.